mlflow/store/artifact/ftp_artifact_repo.py

```diff
@@ -105,7 +105,7 @@ def list_artifacts(self, path=None):
                 return []
             artifact_files = ftp.nlst(list_dir)
             infos = []
-            for file_name in artifact_files:
+            for file_name in artifact_files[2:]:
```

Comment:
Starting from element # 2 sounds dangerous. Can we instead skip if `file_name` is `"."` or `".."`

Reply:
Thank you for the input. I removed the `[2:]` and added a `list(filter(lambda x: x != "." and x != "..", artifact_list))` before to remove any `"."` and `".."`.

Reply:
Hi @mparkhe! Could you give us a follow up on this? Do you think it is relevant to change and merge with MLflow master? Is there anything we can do to help more with the process?